website/docusaurus.config.js

```diff
@@ -71,14 +71,14 @@ var siteSettings = {
     },
     announcementBar: {
       id: "biweekly-demos",
-      content: "Join us on May 14th for the dbt Cloud Launch Showcase event. Discover the latest innovations and watch live demos! ",
+      content: "Join us on May 14th and 15th for the dbt Labs Launch Showcase",
       backgroundColor: "#047377",
       textColor: "#fff",
       isCloseable: true,
     },
     announcementBarActive: true,
     announcementBarLink:
-      "https://www.getdbt.com/resources/webinars/dbt-cloud-launch-showcase",
+      "https://www.getdbt.com/resources/webinars/dbt-cloud-launch-showcase/?utm_medium=internal&utm_source=docs&utm_campaign=q2-2025_dbt-cloud-launch-showcase_aw&utm_content=____&utm_term=all___",
```

Comment:
non-blocking sugg: consider using a URL shortener

Reply:
oh yea it's a utm link that jakki provided in the thread!